Samples/AppLifecycle/StateNotifications/cs-wpf-packaged/CsWpfState/CsWpfState.csproj

```diff
@@ -4,8 +4,8 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
     <UseWPF>true</UseWPF>
-    <Platforms>x64;x86</Platforms>
-    <RuntimeIdentifiers>win10-x64;win10-x86;win-x86;win-x64</RuntimeIdentifiers>
+    <Platforms>x64;x86;ARM64</Platforms>
+    <RuntimeIdentifiers>win10-x64;win10-x86;win10-arm64;win-x86;win-x64;win-arm64</RuntimeIdentifiers>
```

Comment:
win-* RIDs imply execution on Win7, which WinAppSDK doesn't support - should remove throughout all samples

Reply:
I'll do this separately